tests/framework/grid/CheckboxColumnTest.php

```
@@ -33,6 +37,17 @@ protected function setUp(): void
         Yii::$app->assetManager->bundles['yii\web\JqueryAsset'] = false;
     }
 
+    protected function createWidgetInstance(array $config = []): BaseObject
+    {
+        return new CheckboxColumn(array_merge(['grid' => $this->getGrid()], $config));
+    }
+
+    protected function triggerClientScriptDispatch(BaseObject $widget): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$widget'. (undefined)

(UnusedFormalParameter)